From e5ebe63368c8b36310da609c625235647eae4340 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?R=C3=A9mi=20Lef=C3=A8vre?= <rlefevre@dmy.fr>
Date: Tue, 10 Dec 2019 16:51:24 +0100
Subject: testsuite: T12600 avoid broken pipe on Alpine

Use grep -m 1 instead of | head -n 1 to avoid broken pipe.
This should be investigated.
---
 testsuite/tests/simplCore/should_compile/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testsuite/tests/simplCore/should_compile/Makefile b/testsuite/tests/simplCore/should_compile/Makefile
index 5f077b2ebf..b38868bdaf 100644
--- a/testsuite/tests/simplCore/should_compile/Makefile
+++ b/testsuite/tests/simplCore/should_compile/Makefile
@@ -238,7 +238,7 @@ T11272:
 # We expect to see a $wfoo worker that doesn't take any dictionaries.
 .PHONY: T12600
 T12600:
-	'$(TEST_HC)' $(TEST_HC_OPTS) -c -O -ddump-prep -dsuppress-all -dsuppress-uniques -dno-suppress-type-signatures -dppr-cols=200 T12600.hs | grep "wfoo" | head -n 1
+	'$(TEST_HC)' $(TEST_HC_OPTS) -c -O -ddump-prep -dsuppress-all -dsuppress-uniques -dno-suppress-type-signatures -dppr-cols=200 T12600.hs | grep -m 1 "wfoo"
 
 # We don't expect to case match on any literal numbers other than
 # 0 or 1. See T14140.hs for an explanation.
-- 
2.17.1

